frontends/qt/Makefile

```diff
@@ -52,7 +52,7 @@ osx:
 	mv build/BitBox.app build/osx/
 	cp resources/MacOS/Info.plist build/osx/BitBox.app/Contents/
 	cp resources/MacOS/icon.icns build/osx/BitBox.app/Contents/Resources/
-	macdeployqt build/osx/BitBox.app/
```

Comment:
The spurious trailing slash was causing macdeployqt fail on github macos runner.
Not sure where it was coming from but it doesn't seem to be required according to https://doc.qt.io/qt-5/macos-deployment.html.